packages/smb-commons/src/services/events/event.service.test.ts

```diff
@@ -0,0 +1,28 @@
+import { suite } from 'uvu';
+import * as assert from 'uvu/assert';
+
+import { DI, loadProviders, wait } from '../../../test/test.utils';
+import { EventService } from './event.service';
+
+const test = suite('Event Service');
+
+test.before.each((context) => {
+  loadProviders();
+  context.events = DI.get<EventService>('EventService');
+});
+
+test.after.each(() => {
+  DI.unsetAll();
+});
+
+test('should publish and consume events', (context) => {
+  let message;
+  context.events.on('message', (args) => (message = args));
+  context.events.next('message', 'test message');
+
+  wait();
```

Comment:
Although the test is working, here we should `await` the wait and make this test async.

Reply:
True, there is the wait.